redfish-core/lib/log_services.hpp

```diff
@@ -3346,4 +3415,931 @@ inline void requestRoutesPostCodesEntry(App& app)
+                                         "HardwareIsolation.Entry.Type."
+                                         "Critical")
+                        {
+                            entryJson["Severity"] = "Critical";
+                        }
+                        else if ((*severity == "xyz.openbmc_project."
+                                               "HardwareIsolation."
+                                               "Entry.Type.Warning") ||
+                                 (*severity == "xyz.openbmc_project."
+                                               "HardwareIsolation."
+                                               "Entry.Type.Manual"))
+                        {
+                            entryJson["Severity"] = "Warning";
+                        }
+                    }
+                }
+                else if (property.first == "Resolved")
+                {
+                    if (auto resolved = std::get_if<bool>(&property.second))
+                    {
+                        entryJson["Resolved"] = *resolved;
+                    }
+                }
+            }
+        }
+        else if (interface.first == "xyz.openbmc_project."
+                                    "Time.EpochTime")
+        {
+            for (auto& property : interface.second)
+            {
+                if (property.first == "Elapsed")
+                {
+                    if (auto elapsdTime =
+                            std::get_if<uint64_t>(&property.second))
+                    {
+                        entryJson["Created"] = crow::utility::getDateTime(
+                            static_cast<std::time_t>(*elapsdTime));
+                    }
+                }
+            }
+        }
+        else if (interface.first == "xyz.openbmc_project.Association."
+                                    "Definitions")
+        {
+            for (auto& property : interface.second)
+            {
+                if (property.first == "Associations")
+                {
+                    if (auto associations =
+                            std::get_if<AssociationsValType>(&property.second))
+                    {
+                        for (auto& assoc : *associations)
+                        {
+                            if (std::get<0>(assoc) == "isolated_hw")
+                            {
+                                fillIsolatedHwDetailsByObjPath(
+                                    asyncResp,
+                                    sdbusplus::message::object_path(
+                                        std::get<2>(assoc)),
+                                    entryJsonIdx);
+                            }
+                            else if (std::get<0>(assoc) ==
+                                     "isolated_hw_errorlog")
+                            {
+                                sdbusplus::message::object_path errPath =
+                                    std::get<2>(assoc);
```

Comment:
common error 5

Reply:
Agree on the common error 5 but, here I am using for tuple, not for the vector. 
I think tuple won't throw an exception. FYR https://en.cppreference.com/w/cpp/utility/tuple/get - Even that bmcweb common error did not mention for the `tuple` - I am missing anything on this?